spec/unit/installer/xcode/pods_project_generator/target_installer_spec.rb

```diff
@@ -62,6 +62,16 @@ class PodsProjectGenerator
               'AppStore' => 'staticlib',
             }
           end
+
+          it 'embeds swift libraries in Debug mode' do
+            @target.stubs(:product_type).returns(:framework)
+            @installer.send(:add_target).resolved_build_setting('ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES').should == {
+              'Release' => nil,
+              'Debug' => 'YES',
+              'Test' => nil,
```

Comment:
it seems like `Test` would need the fix as well, no? is the fix only applied to `Debug` configurations so that it only applies to unit tests?

Reply:
It was only applied to Debug mostly so it doesn't affect Release mode and app size.  I don't actually know what the Test configuration is typically used for; we don't use that one ourselves.

Reply:
CocoaPods has the concept of build configurations which behave similar to the standard `Debug` & `Release` (can't seem to find any docs on it atm) so I think this would need to be applied to any Debug-like configuration

Reply:
Is there some way to tell which configurations are supposed to behave like Debug?

Reply:
the `debug?` method on `Xcodeproj::Project::Object::XCBuildConfiguration`

Reply:
I updated the implementation to use the `debug?` check, but it still results in `Test` being `nil` for this test